Editor/Window/FleetParametersInput.cs

```diff
@@ -74,8 +74,7 @@ public FleetParametersInput(VisualElement container, ManagedEC2FleetParameters p
             _serverFileButton.RegisterCallback<ClickEvent>(_ =>
             {
                 var value = EditorUtility.OpenFilePanel("Game Server Build File Path (exe)",
-                    _parameters.GameServerFolder,
-                    _parameters.GameServerFile);
+                    _parameters.GameServerFolder, "" );
```

Comment:
The above string says `Game Server Build File Path (exe)`. Is exe the only possible output type? I'm guessing it's different for linux, right?

1. If .exe is not the only type, we should update the string
2. If .exe is the only type, we should instead pass the extension as the third parameter, or alternatively provide all the vaild exensions comma separated since that's allowed.

Thoughts?